src/_includes/services_curated_publications.html

```diff
@@ -1,9 +1,9 @@
-{% assign translations = site.translations[site.lang].curated-publications.softmod %}
+{% assign translations = site.translations[site.lang].curated-publications.[include.service-line] %}
 
-<div class="soft-mod__curated-publications">
+<div class={{include.class}}>
   {%- assign anchor_id = {site.translations[site.lang].software-modernisation.page-sections.publications} -%}
```

Comment:
Let's revisit this file when we're further along in the creation of the **service-line** Jekyl layout. Points to remember for the future.

This file has two responsibilities:
    1. Adding extra markup, an anchor and a class
    1. Passing the service line specific (previously soft mod specific) data into the `curated_publications` include

The first can be moved into the `curated_publications` include. The second can be moved into the individual service line files, so we will likely remove this file later today.

Reply:
These are fixed within [this PR](https://github.com/codurance/site/pull/1819).